Statut HSWro.tex

```diff
@@ -99,122 +97,112 @@ \section{CZŁONKOWIE}
-        \item Oba terminy Walnego Zebrania Członków w trybie nadzwyczajnym muszą być odległe od siebie przynajmniej 3, ale nie bardziej niż 7 dni kalendarzowych.
-      \end{enumerate}
-    \item O terminie i~miejscu Walnego Zebrania Członków Zarząd informuje pocztą elektroniczną wszystkich członków Stowarzyszenia, dołączając do zawiadomienia sprawozdanie z~działalności Stowarzyszenia oraz projekt regulaminu obrad.
-    \item Zarząd jest powołany do kierowania całą działalnością Stowarzyszenia zgodnie z uchwałami Walnego Zebrania Członków.
-    \item W skład Zarządu wchodzi 3 równorzędnych członków.
-    \item Do kompetencji Zarządu należą:
-      \begin{enumerate}
-        \item realizacja celów Stowarzyszenia,
-        \item wykonywanie uchwał Walnego Zebrania Członków,
-        \item sporządzanie planów pracy i budżetu,
-        \item sprawowanie zarządu nad majątkiem Stowarzyszenia,
-        \item podejmowanie uchwał o zarządzaniu majątkiem Stowarzyszenia,
-        \item reprezentowanie Stowarzyszenia na zewnątrz,
-        \item zwoływanie Walnego Zebrania Członków,
-        \item przyjmowanie i skreślanie członków zwyczajnych,
-        \item składanie sprawozdań ze swojej działalności na Walnym Zebraniu Członków,
-        \item sporządzanie rocznego sprawozdania finansowego,
-        \item zwalnianie w uzasadnionych przypadkach Członków Zwyczajnych z obowiązku płacenia składek w całości bądź w części.
-      \end{enumerate}
-    \item Posiedzenia Zarządu odbywają się w miarę potrzeb, nie rzadziej jednak niż raz na rok.
-    \item Posiedzenia Zarządu zwołuje:
-      \begin{enumerate}
-        \item dwóch członków zarządu działających łącznie,
-        \item Komisja Rewizyjna.
-      \end{enumerate}
-    \item Komisja Rewizyjna powoływana jest do sprawowania kontroli nad działalnością Stowarzyszenia.
-    \item Komisja Rewizyjna składa się z 3 równorzędnych członków.
-    \item Do kompetencji Komisji Rewizyjnej należy:
-      \begin{enumerate}
-        \item kontrolowanie działalności Zarządu,
-        \item składanie sprawozdań z kontroli na Walnym Zebraniu Członków,
-        \item prawo wystąpienia z wnioskiem o zwołanie Walnego Zebrania Członków oraz zebrania Zarządu,
-        \item składanie wniosków o absolutorium dla Zarządu Stowarzyszenia,
-        \item składanie sprawozdań ze swojej działalności na Walnym Zebraniu Członków,
-        \item zatwierdzanie rocznego sprawozdania finansowego.
-      \end{enumerate}
+    \item Kadencja wszystkich organów wybieralnych Stowarzyszenia trwa rok i~jest wspólna.
+    \item Uchwały wszystkich organów Stowarzyszenia zapadają zwykłą większością głosów przy obecności co najmniej połowy członków uprawnionych do głosowania, chyba że dalsze postanowienia statutu stanowią inaczej.
+    \item Uchwały o wyborze i odwoływaniu organów stowarzyszenia, zmianach statutu oraz rozwiązania stowarzyszenia zapadają większością 2/3 głosów, przy obecności co najmniej połowy członków uprawnionych do głosowania.
```

Comment:
Według mnie może być duży problem, kiedy ludzie po prostu przestaną przychodzić na zebrania, albo nie będą mieli czasu.

Ja skleciłem coś takiego:
> \begin{art}
>   \begin{para}
>     Do składu zebrania wlicza się:
>     \begin{enumerate}
>       \item{Zdolnych obecnych na posiedzeniu}
>       \item{Zdolnych deklarujących nieobecność, poza wyjątkami.}
>     \end{enumerate}
>   \end{para}
> 
>   \begin{para}
>     Do składu zebrania nie wlicza się:
>     \begin{enumerate}
>       \item{Zdolnych, którzy nie zadeklarowali nieobecności w czasie 7 dni od ogłoszenia zebrania.}
>       \item{Zdolnych, którzy zadeklarowali nieobecność, lecz zażyczyli sobie o nie wliczanie do składu.}
>     \end{enumerate}
>   \end{para}
> \end{art}

Reply:
ale nie ma ze nie przychodzisz. To nie jest sporg fikcyjy a podmiot prawny. Chcesz by caly majatek czy marke stowarzyszenia hackerspace przejelo jakies kolo studenckie co swoich czlonkow wprowadzi i skorzysta na zebraniu co mialo mala frekwencje? Takie akcje sie zdarzaly w stowarzyszeniach. Raz do roku musisz cisnac ludzi by byli lub przekazali swoj glos. Walne zebranie nie odbywa sie co tydzien a raz do roku.